kinode/packages/app_store/app_store/src/lib.rs

```diff
@@ -313,14 +120,7 @@ fn init(our: Address) {
 /// function defined for each kind of message. check whether the source
 /// of the message is allowed to send that kind of message to us.
 /// finally, fire a response if expected from a request.
-fn handle_message(
-    our: &Address,
-    state: &mut State,
-    eth_provider: &eth::Provider,
-    requested_apis: &mut HashMap<PackageId, RequestedPackage>,
-    requested_packages: &mut HashMap<PackageId, RequestedPackage>,
-    message: &Message,
-) -> anyhow::Result<()> {
+fn handle_message(state: &mut State, message: &Message) -> anyhow::Result<()> {
     match message {
```

Comment:
Formatting nit: do we want to change this to use the
```
if message.is_request() {
}
```
pattern for less indentation?